lib/widgets/nutrition/nutritional_plan_detail.dart

```diff
@@ -205,6 +210,17 @@ class NutritionalPlanDetailWidget extends StatelessWidget {
             ),
           ),
           const Padding(padding: EdgeInsets.all(8.0)),
+          Text(
+            'Planned / Logged / 7 Day Average',
```

Comment:
can you mark this for translation? You just need to add them to app_en.arb

Reply:
I would translate this as three different terms, then we can use them somewhere else in the app (logged should be available already I think)